migration/sql-files/068-spaces_identities_fk.sql

```diff
@@ -0,0 +1,8 @@
+-- add a foreign key from spaces.owner_id to identities.id
+alter table spaces add constraint spaces_owner_id_identities_id_fk foreign key (owner_id) REFERENCES identities (id);
+-- add index on spaces.name
+create index idx_spaces_name on spaces (lower(name));
```

Comment:
This index is not required in this case, as you already hit spaces_name_idx if you use lower(name) = lower('')